### PR TITLE
Encrypt local VCs

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: lib/encrypt.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+// PouchDB.debug.enable('pouchdb:find');
+
+PouchDB.plugin(pouchFind);
+
+export class EncryptedDb {
+  constructor({db, dbName, edv, invocationSigner}) {
+    this.edv = edv;
+    this.invocationSigner = invocationSigner;
+    this.db = db || new PouchDB(dbName);
+    // TODO: This needs to handle encrypted indexes.
```
Comment (JoshDunnDev) [inline lib/encrypt.js:19]: Issue has been created for this: https://github.com/digitalbazaar/bedrock-web-local-vc-store/issues/10

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
looks good. would it be possible to add more tests?

**Review by dlongley — COMMENTED** [review]
There's a lot here and I haven't had time to really do an in depth review (and don't think that I will). So +1 for relying on more tests here.

**Comment by JoshDunnDev** [comment]
I will work on adding tests to this library.

File: tests/package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "test": "npm run test-node && npm run test-karma",
+    "test-node": "mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require ./test-mocha.js ./*.spec.js",
+    "test-karma": "karma start ../karma.conf.js",
+    "test-watch": "mocha -r esm --watch --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require ./test-mocha.js ./*.spec.js",
+    "coverage": "nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report"
+  },
+  "dependencies": {
+    "@digitalbazaar/did-method-key": "^1.1.0",
```
Comment (dmitrizagidulin) [inline tests/package.json:15]: Reminder to update the did-method-key dep to 2.0, and signature lib to 3.0

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by dmitrizagidulin** [comment]
Published `bedrock-web-local-vc-store@3.0.0`.

**Pull request merged**